<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>628</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="main" stretch="1,3">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>80</number>
    </property>
    <property name="topMargin">
     <number>80</number>
    </property>
    <property name="rightMargin">
     <number>80</number>
    </property>
    <property name="bottomMargin">
     <number>80</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="left_side" stretch="0,0,0,0,0,0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="logo">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../resources/ceva_logo.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="username_label">
        <property name="text">
         <string>Meter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="add_employee_button">
        <property name="text">
         <string>Pridať zamestnanca</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_employee_button">
        <property name="text">
         <string>Vymazať zamestnanca</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_instruction_button">
        <property name="text">
         <string>Pridať inštrukciu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="histogram_button">
        <property name="text">
         <string>Histogram</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logout_button">
        <property name="text">
         <string>Odhlásiť sa</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="right_side">
      <item>
       <layout class="QHBoxLayout" name="upper_search">
        <item>
         <widget class="QLineEdit" name="search_input">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="placeholderText">
           <string>Vyhľadať</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
